Rakefile

```diff
@@ -36,6 +36,7 @@ if Bundler.rubygems.find_name('github_changelog_generator').any?
     raise "Set CHANGELOG_GITHUB_TOKEN environment variable eg 'export CHANGELOG_GITHUB_TOKEN=valid_token_here'" if Rake.application.top_level_tasks.include? "changelog" and ENV['CHANGELOG_GITHUB_TOKEN'].nil?
     config.user = "#{changelog_user}"
     config.project = "#{changelog_project}"
+    config.since_tag = '2.4.0'
```

Comment:
Hi @RandomNoun7 Changes looks good.
since_tag can be updated in the .sync.yaml file and we can run pdk update.
Not sure if we need to update since_tag since the module has been using the Changelog generator from release 3.0.0.
Let me also try to run.